step2-01/demo/src/types/index.ts

```diff
@@ -47,13 +47,18 @@ let choose1 = <Specific1>{ common: '5' };
 type CatStatus = 'alive' | 'dead' | 'both';
 
 // Classes
-class Animal {}
+class Animal { }
 
 // Illustration purposes only
 // In real apps, avoid inheritance if possible
 // noted exception: React.Component with react@<16.8.0
-class Cat extends Animal {}
-class Dog extends Animal {}
+class Cat extends Animal { }
+class Dog extends Animal { }
 
-// adding an export to turn this into a "module"
+// Any Type - avoid if possible
```

Comment:
I think this is worth covering, but hopefully the example conveys that it's not encouraged... :)